File: Array/fighting_the_darkness.cpp
// solution is in class Solution


//Initial template for C++

#include <bits/stdc++.h>
using namespace std;

 // } Driver Code Ends
//User function template for C++

class Solution{   
public:
    int maxDays(int arr[],int n){
        // code here 
        int max = arr[0];
        for(int i=1; i<n; i++){
            if(max<arr[i])
                max=arr[i];
        }
        return max;
    }
};

// { Driver Code Starts.

int main() 
{
    int t;
    cin >> t;
    while (t--)
    {
        int n;
        cin>>n;
        int arr[n];
        for(int i=0;i<n;i++)
            cin>>arr[i];
        Solution ob;
        cout << ob.maxDays(arr, n) << "\n";
    }
    return 0;
}  // } Driver Code Ends

File: Array/fighting_the_darkness.java
// solution is in class Solution


//Initial Template for Java

import java.io.*;

class fighting_the_darkness{
    public static void main(String[] args) throws IOException 
    {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int tc = Integer.parseInt(br.readLine());

        while (tc-- > 0) {
            int n = Integer.parseInt(br.readLine());
            String line1 = br.readLine();
            String[] a1 = line1.trim().split("\\s+");
            long arr[] = new long[n];
            for (int i = 0; i < n; i++) {
                arr[i] = Long.parseLong(a1[i]);
            }
            Solve ob = new  Solve();
            System.out.println(ob.maxDays(arr,n));
        }
    }
}
// } Driver Code Ends


//User function Template for Java

class Solve{
    long maxDays(long arr[], int n){
        // code here 
        long max = arr[0];
        for(int i=1; i<n; i++){
            if(max<arr[i])
                max=arr[i];
        }
        return max;
    }
}